### ISSUE TITLE
Documentation for ADW does not indicate the need for BASE_URL in .env

### ISSUE THREAD

**Issue by binduwavell** [Alfresco/alfresco-content-app#2558]
For both ADW 2.8.0 and 2.9.0 there is no documentation that indicates you must provide a value for BASE_URL. In our case we are not using APS, so in theory this should not be required.

At least the docs should be updated. Or possibly a better solution would be to update the `proxy.*conf.js` files to use `APP_CONFIG_BPM_HOST` in the event `BASE_URL` is not provided. At least I *think* these need to effectively be the same?

**Comment by binduwavell** [comment]
I should add that this is required during development and not once you package/deploy.

**Comment by montgolfiere** [comment]
@binduwavell Thanks for the feedback. Can you also add links to the docs that you are using ( cc @oliveralfresco2018 ) ?

In case it helps, you could also look at the ACS Deployment docker-compose for a very basic / vanilla setup, eg.

Alfresco Content App
- https://github.com/Alfresco/acs-deployment/blob/master/docker-compose/community-docker-compose.yml#L114-L118

Alfresco Digital Workspace
- https://github.com/Alfresco/acs-deployment/blob/master/docker-compose/docker-compose.yml#L148-L154

**Comment by binduwavell** [comment]
Thank you @montgolfiere for the reply. The docs I'm referencing are those included in the ADW source zip file I downloaded from the Alfresco Nexus server. 

There are docs in bot content-ce and content-ee and some various README files. 

I had not seen Angel's post on ADW (I did see the post on ACA a month or two ago). I published something similar to the beginning of his ADW post a couple days before him: https://www.ziaconsulting.com/technology/alfresco-technology/getting-started-with-alfresco-digital-workspace-development/ :)
